### ISSUE TITLE
Exception loading list index from file

### ISSUE THREAD

**Issue by itaim** [jerryjliu/gpt_index#484]
Exception loading saved list index with version 0.4.6
Loads okay with 0.4.4
```
return cls.load_from_string(file_contents, **kwargs)
  File "/Users/***/chatbot/.venv/lib/python3.10/site-packages/gpt_index/indices/base.py", line 431, in load_from_string
    index_struct = docstore.get_document(index_struct_id)
  File "/Users/***/chatbot/.venv/lib/python3.10/site-packages/gpt_index/docstore.py", line 97, in get_document
    raise ValueError(f"doc_id {doc_id} not found.")
ValueError: doc_id 88151a8e-e3c5-41f2-bb14-f7f6cb9c4762 not found.
```

**Comment by jerryjliu** [comment]
@itaim we refactored the docstore in 0.4.5, it's possible that introduced some breaking changes i wasn't aware of. 

 could you try regenerating the list index + saving + loading again to see if it's an issue? (the cost should be 0, list index doesn't make any LLM / embedding calls)

**Comment by jerryjliu** [comment]
also i think i just figured out the issue, will create a patch fix (but also let me know if regenerating works)

**Comment by itaim** [comment]
It has ~30k documents so it would take a while to re-embed. I can try to load the json, parse and rebuild but if you already figured it out I'' wait for the fix 🙂

**Comment by jerryjliu** [comment]
@itaim should be fixed with #485 , give it a shot

**Comment by itaim** [comment]
Works. thank you!
